### PR TITLE
feat(todo-mvc): polish up and add i18n

### PR DESCRIPTION
**Pull request by juliusmarminge**
Closes #

## 🎯 Changes

What changes are made in this PR? Is it a feature or a bug fix?

- polishes up the example
  - implements the `toggle all` button
  - move some stuff out to make the trpc parts easier to navigate
- adds i18n
  - bit of a work in progress... getting some weird flashes cause trpc reverts the chosen locale cause it only checks the `accept-language` header now, needs a better parser there so that it can be overridden 
  

## ✅ Checklist

- [ ] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

File: examples/next-prisma-todomvc/README.md [hunk 1]
```diff
@@ -2,32 +2,22 @@
 
 - [TodoMVC](https://todomvc.com/) implemented with tRPC + [Prisma](https://prisma.io)
 - Live at [todomvc.trpc.io](https://todomvc.trpc.io)
-- Try in CodeSandbox: [https://githubbox.com/trpc/trpc/tree/main/examples/next-prisma-todomvc](https://codesandbox.io/s/github/trpc/trpc/tree/main/examples/next-prisma-todomvc?file=/pages/%5Bfilter%5D.tsx)
```
Comment (juliusmarminge) [inline examples/next-prisma-todomvc/README.md:5]: this was broken so removed it

File: examples/next-prisma-todomvc/README.md [hunk 2]
```diff
@@ -2,32 +2,22 @@
...
 ```
 
-### ℹ️ This example contains _both_ Postgres _and_ SQLite
-
-- This project contains both Postgres & SQLite in able for us to easily showcase it in both CodeSandbox and deploy on Vercel.
-- This is not a recommended approach. If you want to fork this project as a starter, do the following:
-  - `rm -rf ./prisma/_sqlite`
-  - Update `package.json`:
-    - remove all references of `sqlite`
-    - replace commands using `*:sqlite` with `*:postgres`
```
Comment (juliusmarminge) [inline examples/next-prisma-todomvc/README.md:29]: the sqlite version  was removed a while ago

File: examples/next-prisma-todomvc/prisma/migrations/20230322185952_/migration.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+-- RenameIndex
+ALTER INDEX "Task.createdAt_unique" RENAME TO "Task_createdAt_key";
```
Comment (juliusmarminge) [inline examples/next-prisma-todomvc/prisma/migrations/20230322185952_/migration.sql]: got this when starting dev - have we changed the schema without running migrations earlier??

File: examples/next-prisma-todomvc/src/pages/[filter].tsx [hunk 4]
```diff
@@ -238,150 +232,97 @@ export default function TodosPage({
...
-                Completed
-              </Link>
-            </li>
+            {filters.map((filter) => (
+              <li key={'filter-' + filter}>
+                <Link
+                  href={'/' + filter}
+                  className={filter == props.filter ? 'selected' : ''}
+                >
+                  {t(filter)[0].toUpperCase() + t(filter).slice(1)}
```
Comment (juliusmarminge) [inline examples/next-prisma-todomvc/src/pages/[filter].tsx:275]: god i miss my tailwind `capitalize` class 🤣

File: examples/next-prisma-todomvc/src/pages/_app.tsx [hunk 5]
```diff
@@ -1,8 +1,39 @@
...
+  const passedProps = {
+    ...props,
+    pageProps: {
+      ...props.pageProps,
+      ...i18n,
+    },
+    router: locale ? { locale } : props.router,
+  } as unknown as ComponentProps<typeof I18nextAdapter>;
+  return <I18nextAdapter {...passedProps} />;
+};
```
Comment (juliusmarminge) [inline examples/next-prisma-todomvc/src/pages/_app.tsx:28]: shamelessly stolen from cal.com

**Review by KATT — APPROVED** [review]

**Review by sachinraja — DISMISSED** [review]

**Review by sachinraja — DISMISSED** [review]

**Review by sachinraja — APPROVED** [review]

**Pull request merged**